src/_includes/banner.html

```diff
@@ -1,7 +1,7 @@
 {% comment %}Drop the div .alert classes to revert to the default banner styling.{% endcomment -%}
 <div class="banner">
   <p class="banner__text">
-    Join Dart and Flutter at Google I/O 2022 live from Shoreline Amphitheatre and online May 11-12. <a href="https://io.google/2022/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_svd&utm_content=">Register now</a><br>
+    <a href="https://io.google/2022/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_svd&utm_content=">Google I/O is happening now. Join us.</a><br>
```

Comment:
Maybe remove the final period to match our other banner links:

```suggestion
    <a href="https://io.google/2022/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_svd&utm_content=">Google I/O is happening now. Join us</a><br>
```